config/config/sc-config.yaml

```diff
@@ -902,6 +902,18 @@ opensearch:
     nodeSelector: {}
     tolerations: []
 
+  promIndexAlerts:
+    - pattern: kubernetes-default
```

Comment:
I'm guessing that the the name _pattern_ here is a bit ambiguous to what we have used elsewhere, like in the curation retention or opensearch roles where we usually have a `*` following the index name.

Reply:
true, but I didn't want to add them here. just to trim the * in the alert.
do you have another idea?

Reply:
Maybe add a comment/example saying that a wildcard is added to the end of the given string

Reply:
Or rename it to prefix perhaps, just to make it more clear?

Reply:
prefix makes sense 👍🏻

Reply:
@aarnq and @OlleLarsson I hope the new name works better